frappe/tests/ui/test_kanban/test_kanban_column.js

```diff
@@ -0,0 +1,22 @@
+QUnit.module('views');
+
+QUnit.test("Test setting colour for column [Kanban view]", function(assert) {
+	assert.expect(3);
+	let done = assert.async();
+
+	frappe.run_serially([
+		() => frappe.tests.setup_doctype('Kanban Board'),
+		() => frappe.set_route("List", "ToDo", "Kanban", "kanban 1"),
+		() => frappe.timeout(0.5),
+		() => assert.deepEqual(frappe.get_route(), ["List", "ToDo", "Kanban", "kanban 1"], "Kanban view opened successfully."),
+		//set colour for columns
+		() => frappe.tests.click_and_wait('div:nth-child(3) > div > div > ul > li > div.red'),
```

Comment:
Columns position might change, select the column using it's name and then change the color